inc/planningrecall.class.php

```diff
@@ -53,7 +53,7 @@ static function canCreate() {
 
 
    function canCreateItem() {
-      return $this->fields['users_id'] == Session::getLoginUserID();
+      return true;
```

Comment:
Hum... Not sure for this one. Should we not look for some rights here, ping @orthagh

Reply:
regarding #6234, this make sense, but i think the both pr are wrong.

In the relative notification, only the task creator or the assignee can be recipient.
So only them should be allowed to add a recall.

For other users, planning recalls dropdown should not be displayed (or maybe in read-only, but i'm not sure it's needed).
Other scenario like a supervisor adding recall for users but not creator nor assignee requires a new right to do it, so an evolution not a bug.

Reply:
i revert my comment, 

Recipient Request retrieved from the planning recall not from the task.
So the current scenari is anybody who see the task can add a recall on it to be notified himself.

So the previous code changed by this pr is the good one and #6234 is wrong also